Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.66.0"
+version = "1.66.1"
```

this isn't a bug fix, so should be 
```suggestion
version = "1.67.0"
```